Source/Core/Core/HW/WiimoteEmu/EmuSubroutines.cpp

```diff
@@ -119,11 +119,12 @@ void Wiimote::HidOutputReport(const wm_report* const sr, const bool send_ack)
 
   case RT_WRITE_DATA:  // 0x16
     WriteData(reinterpret_cast<const wm_write_data*>(sr->data));
+    return;  // sends its own ack
     break;
```

Comment:
Does our code style want _both_  `return` and `break` or should the `break` be removed here?

Reply:
return; break; maintains consistency. I don't know what the guidelines say.

Reply:
More of a general question, I tend to err on the consistency side as well though.